src/pages/DisplayCards.tsx

```diff
@@ -31,9 +31,32 @@ const initialRightCard: Card = {
   isEditing: false,
 };
 
+const CurrentDimension = 0;
+
 const DisplayCards: React.FC = () => {
   const [leftState, setLeftState] = useState(initialLeftCard);
   const [rightState, setRightState] = useState(initialRightCard);
+
+  useEffect(() => {
+    fetch("/api/cards")
+      .then((res) => res.json())
+      .then((res) =>
+        setLeftState({
+          text: res[CurrentDimension].Statement,
+          isEditing: false,
+        })
+      );
+
+    fetch("/api/cards")
```

Comment:
You don't need to make a second fetch request to get the cards' data again. You should already receive it in the first request and should be able to set the Right card state there directly, just as you did with the left card state. Another solution may just be to store the array of cards as state and not to separate them out. This may be especially good in cases when a dimension has three cards (if that can even happen?)